libs/community/langchain_community/embeddings/gpt4all.py

```diff
@@ -22,7 +22,7 @@ class GPT4AllEmbeddings(BaseModel, Embeddings):
             )
     """
 
-    model_name: str
+    model_name: Optional[str] = "all-MiniLM-L6-v2-f16.gguf"
```

Comment:
```suggestion
    model_name: Optional[str] = None
```

Theres no need to force the usage of this model at the GPT4AllEmbeddings level since it's already covered in the Embed4All class.